content/blog/2022-07-05-serving-models-with-mlem.md

```diff
@@ -0,0 +1,283 @@
+---
+title: Serving Machine Learning Models with MLEM
+date: 2022-07-05
+description: >
+  Once you have a machine learning model that's ready for production, getting it
+  out can be complicated. In this tutorial, we're going to use MLEM to deploy a
+  model to an API.
+descriptionLong: >
+  Getting a machine learning model out to our end users can be an involved task.
+  You have to make sure it's in the correct format for the environment and for
+  the way it will be consumed. We're going to walk through how you can use MLEM
+  to save your model and serve it with a web API.
+picture: 2022-07-05/serve_mlem_models.png
+pictureComment: Serving Machine Models with MLEM
+author: milecia_mcgregor
+commentsUrl: https://discuss.dvc.org/t/syncing-data-to-aws-s3/1192
+tags:
+  - MLOps
+  - DVC
+  - Git
+  - MLEM
+  - Collaboration
+---
+
+Training a machine learning model is only one step in the process of getting
+something useful out to end-users. There are many goals for machine learning
+projects and we need accurate models that provide us with information based on
+the data received from a production environment. That could mean data coming
+from an API or batch processing, for example.
```

Comment:
Minor, but I think something like this makes the ideas in the different sentences more cohesive:
```suggestion
Training a machine learning model is only one step in the process of getting
something useful out to end-users. When time comes to _productionize_ it, no one
approach fits-all. That's because there are many goals for ML projects; That could
mean to respond to real-time queries coming from an API or batch processing
predictions, for example.
```
Although "no one approach fits-all" is kind of what MLEM tries to address so I'm not sure about this exact phrasing.